rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/InfoView.java

```diff
@@ -451,4 +471,21 @@ public void hostSelected(Host h) {
         this.selNS = null;
         this.selNode = null;
     }
+
+    private String[] splitAndExtractNameAndParamsFromDescription(String input) {
+        String[] elements = input.split("[,\\s]+");
```

Comment:
you can make this code much simpler

return  input.split("[,\\s]+", 2);

Reply:
Very nice !